canonical-webteam/documentation-builder
# Fix the warning notifications

## Done
Updated the annotation extension to use `p-notification--caution` instead of the deprecated `p-notificaiton--warning`.

## QA
- In master branch, open `docs/en/index.md`
- Add the line:
``` markdown
!!! Warning:
    A machine destined for MAAS will have its disk space overwritten.


!!! Information:
    Some information
```
- Run:
``` bash
python3 -m venv env3 && source env3/bin/activate  # Create encapsulated environment
pip install -e .  # Install the module in editable mode

documentation-builder --source-folder docs  # build the documentation-builder's own documentation
xdg-open build/en/index.html  # Open up the documentation page
```
- See that the warning is not styled
- See that there is no information notifications
- Check out this branch
- Run `documentation-builder --source-folder docs`
- Refresh the docs page and see the notification is styled
